docs/user/autoyast.md

```diff
@@ -372,6 +437,24 @@ of services to enable and a list of services to disable.
 | services.disable  | Planned   |       |         |
 | services.ondemand | Planned   |       |         |
 
+At this point, it is possible to use post-scripts to enable/disable services.
```

Comment:
I think it should be possible to use it also to install additional software, right?

Reply:
Yes, that's another use case. I think you need to move the `zypp.pid` file, but it is possible (just like with AutoYaST).